ompi/mca/coll/libnbc/nbc.c

```diff
@@ -655,15 +655,15 @@ int NBC_Start(NBC_Handle *handle) {
   }
 
   /* kick off first round */
-  handle->super.req_state = OMPI_REQUEST_ACTIVE;
-  handle->super.req_status.MPI_ERROR = OMPI_SUCCESS;
+  handle->super.super.req_state = OMPI_REQUEST_ACTIVE;
+  handle->super.super.req_status.MPI_ERROR = OMPI_SUCCESS;
   res = NBC_Start_round(handle);
   if (OPAL_UNLIKELY(OMPI_SUCCESS != res)) {
     return res;
   }
 
   OPAL_THREAD_LOCK(&mca_coll_libnbc_component.lock);
-  opal_list_append(&mca_coll_libnbc_component.active_requests, &(handle->super.super.super));
+  opal_list_append(&mca_coll_libnbc_component.active_requests, &(handle->super.super.super.super));
```

Comment:
Oh my, this is getting ugly ;) It would be nicer if we simply cast it to opal_list_item_t directly.